src/pages/chainlink-functions/tutorials/api-use-secrets-offchain.md

```diff
@@ -0,0 +1,277 @@
+---
+layout: ../../../layouts/MainLayout.astro
+section: chainlinkFunctions
+date: Last Modified
+title: "Using Off-chain Secrets in Requests"
+setup: |
+  import ChainlinkFunctions from "@features/chainlink-functions/common/ChainlinkFunctions.astro"
+---
+
+This tutorial shows you how to leverage the Off-chain Secrets capability to share encrypted secrets off-chain via HTTP with a Decentralized Oracle Network.
+Off-chain secrets are encrypted, users decide where to host them (e.g., aws S3, google drive, or IPFS), and fetched by the Decentralized Oracle Network via HTTP(s).
```

Comment:
```suggestion
This tutorial shows you how share encrypted secrets off-chain with a Decentralized Oracle Network (DON) via HTTP. Off-chain secrets are encrypted and stored on AWS S3, Google Drive, IPFS, or any other service where the DON can fetch them via HTTP.
```
- Simplify the first sentence. The off-chain secrets capability isn't as important as just stating 
- Include (DON) in the first instance where we use the acronym in the doc.
- Avoid e.g. and asides. Usually you can split the sentence into two sentences.
- Capitalize AWS and Google products that are proper nouns.